spec/unit/select_spec.rb

```diff
@@ -400,7 +400,136 @@ def output_helper(prompt, choices, active, options = {})
     prompt.input.rewind
 
     expect {
-      prompt.select('What size?', choices, default: 10)
+      prompt.select("What size?", choices, default: 10)
     }.to raise_error(TTY::Prompt::ConfigurationError, /`10` out of range \(1 - 3\)/)
   end
+
+  context "with filter" do
```

Comment:
I would encourage you to use `output_helper` to make the specs more consistent. Especially when it comes down to symbols for select menus as these currently fail on Windows.